src/components/playbackSettings/playbackSettings.template.html

```diff
@@ -180,6 +180,17 @@ <h2 class="sectionTitle">
             <div class="fieldDescription checkboxFieldDescription">${EnableTrueHdHelp}</div>
         </div>
 
+        <div class="selectContainer">
+            <select is="emby-select" id="selectPreferredTranscodeVideoCodec" label="${LabelSelectPreferredTranscodeVideoCodec}">
+                <option value="">${Auto}</option>
+                <option value="h264">H264</option>
+                <option value="hevc">HEVC</option>
+                <option value="av1">AV1</option>
+                <option value="vp9">VP9</option>
```

Comment:
Transcoding to VP9 is not well supported and tested. Also it's not widely available in hardware encoder. IMO there's no need to show it.